invenio_communities/config.py

```diff
@@ -35,9 +35,12 @@
 COMMUNITIES_DELETE_HOLDOUT_TIME = timedelta(days=365)
 """Time after which the communities marked for deletion are hard-deleted."""
 
-COMMUNITIES_LOGO_EXTENSIONS = ['png', 'jpg', 'jpeg']
+COMMUNITIES_LOGO_EXTENSIONS = ['png', 'jpg', 'jpeg', 'svg']
 """Allowed file extensions for the communities logo."""
 
+COMMUNITIES_LOGO_MAX_SIZE = 1000 * 1000 * 1.5  # 1.5 MB
```

Comment:
Yes, but it would be just his wrong assumption about what is 1.5 MB ;)
```
$ du output.dat -h                                                                                                                                                                                                              
1.5M	output.dat
                                                                                                                                                                                                                                   
$ du output.dat -h --si                                                                                                                                                                                                         
1.6M	output.dat
```

I understand the benefits of having size definitions in base 2 on the technical side of the application, but this is a user-facing size definition, and almost all user-facing size definitions in services like Dropbox, Google Drive etc. use SI units. I would argue that using "MiB" on the front-end is more confusing than the corner case you have in mind.

TDB tomorrow.